scripts/deleteWorkout.js

```diff
@@ -0,0 +1,12 @@
+const DeleteButton = (onClick) => {
+
+    const deleteButton = document.createElement("button");
+    deleteButton.innerText = "delete";
+    deleteButton.classList.add("deleteButton")
```

Comment:
```suggestion
    deleteButton.classList.add("delete-button")
```
Classes seguem o parão kebab-case